project.clj

```diff
@@ -100,6 +100,7 @@
 
   :profiles {:dev {:source-paths  ["dev"]
                    :dependencies  [[org.clojure/tools.namespace "0.2.5"]
+                                   [org.clojure/java.jmx "0.2.0"]
```

Comment:
I don't see any output for the test jar in it, no.

I also tried just putting the `[org.clojure/java.jmx "0.2.0"]` in as a first level dependency in the tk-jetty9 project.clj.  When I `lein install` that to my .m2 repo and try to reference it from a local puppet-server repo `lein test` run, though, I get the same error.  Shouldn't that at least have worked?